docs/source/federation/federation-spec.md

```diff
@@ -282,7 +282,7 @@ The `@requires` directive is used to annotate the required input fieldset from a
 ```graphql
 # extended from the Users service
 extend type User @key(fields: "id") {
-  id: ID!
+  id: ID! @external
```

Comment:
Yup, because the `id` a field that is implemented in another service! 😄 

(Just hoping that this comment will help those following along at home.)